index.html

```diff
@@ -1,207 +1,107 @@
-                                euismod in, pharetra a, ultricies in, diam. Sed arcu. Cras consequat
-                           </div>
-                        </div>
-                    </div>
-                    
-                    <div class="perm-one">
-                        <div class="images">
-                            <img src="images/pic five.jpeg" alt="" class="image-one">
-                        </div>
-                        <div class="names">
-                            <p class="individual">Pesus Darko</p>
-                            <p class="from">United States of America</p>
-                            <div class="content">Morbi in sem quis dui placerat ornare. Pellentesque odio nisi,
-                                euismod in, pharetra a, ultricies in, diam. Sed arcu. Cras consequat
-                           </div>
-                        </div>
-
-                    </div>
-
-                    <div class="perm-one">
-                        <div class="images">
-                            <img src="images/pic five.jpeg" alt="" class="image-one">
-                        </div>
-                        <div class="names">
-                            <p class="individual">William Bamfo</p>
-                            <p class="from">United States of America</p>
-                            <div class="content">Morbi in sem quis dui placerat ornare. Pellentesque odio nisi,
-                                euismod in, pharetra a, ultricies in, diam. Sed arcu. Cras consequat
-                           </div>
-                        </div>
-                    </div>
-
-                    
-                  
-                   
-
+            </article>
+        </section>
+        <section class="feature-in">
+            <div class="container">
+              <h3 class="feature-heading">Sponsors</h3>
+                <div class="logos">
+                   <i class="fa fa-bank"></i>
+                   <i class="fa fa-coffee"></i>
+                   <i class="fa fa-futbol-o"></i>
+                   <i class="fa fa-laptop"></i>
                 </div>
-            </section>
-            <div class="sponsors">
-                <div class="title">Sponsors</div>
-                <div class="ruler"></div>
-
-                <i class="fa fa-bank"></i>
-                <i class="fa fa-coffee"></i>
-                <i class="fa fa-futbol-o"></i>
-                <i class="fa fa-laptop"></i>
             </div>
-            <div class="right">
-                
-                <p class="rite"><span><i class="fa fa-copyright"></i></span> Creative Commons Korea. Some Rights Reserved.All content on the website,
-                 unless otherwise specified, is available under a Creative Commons Attribution 2.0 Korea License.
-                </p>
-            </div>
-
-        </div> 
-       
-        
-
-        </div> 
-
-
-    </div>
-    
-</body>
-</html>
\ No newline at end of file
+        </section>
```

Comment:
- [ ] Kindly correct the "Partner" section, use the same color in the background, there are five business logos, no icons.
- [ ] Please 🙏🏽 correct the position of the "Partner" section, it is below the "Feature Speakers" section.

| Original design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/198764258-43d7eeca-9882-4457-9bb3-cefacbcc1a51.png) | ![image](https://user-images.githubusercontent.com/58642949/198764473-9b4328d6-1a76-40d8-b43a-19709a793e78.png) |